xml/src/main/resources/ehcache-core.xsd

```diff
@@ -260,6 +260,13 @@
           </xs:documentation>
         </xs:annotation>
       </xs:element>
+      <xs:element name="resilience" type="ehcache:fqcn-type" minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation xml:lang="en">
+            Configures the resilience strategy used when the cache's underlying storage fails.
```

Comment:
cache's? Not `cache underlying storae`?

Reply:
Yes. Possessive because it's the underlying storage for the cache.

Reply:
Ok... That's something I never fully understood in English. At what point a thing can possess a thing...

Reply:
I *think* that your version is correct too but my english grammar is intuitive and not learned so I can't even tell you why mine reads better (to me).